src/components/Card/Cards.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Activity from './Activity'
+import styled from 'styled-components/macro'
+import Category from './Category'
+
+export default function Cards({ activities, categories }) {
+  return categories.map(category => (
+    <CardStyled key={category}>
```

Comment:
A category can be twice or more times in a list of cards so it is not a unique key. Maybee you can take uid or the current time in miliseconds.